db/seeds.rb

```diff
@@ -68,14 +68,42 @@
   })
 end
 
+Parent.destroy_all
 puts "Creating Parents..."
 
+Kid.destroy_all
+puts "Creating Kids..."
+
 10.times do |count|
   user = User.create!({
     first_name: Faker::Lovecraft.deity,
     last_name: "of #{Faker::Lovecraft.location}",
     email: "#{Faker::Lovecraft.word}#{count * rand(7) + 1}@campers.com",
     password: "password",
     role: "parent"
+    })
+
+    parent = Parent.create!({
+      user_id: user.id,
+      address: "123 Street",
+      phone_number: "(514) 000-1111"
```

Comment:
Set this to a simple string of numbers. I believe Rails has a method like to_phone that will automatically convert this into a pretty phone string.